app/views/form/review.html.erb

```diff
@@ -1,9 +1,18 @@
-<% content_for :title, "Review lettings log" %>
-<% content_for :breadcrumbs, govuk_breadcrumbs(breadcrumbs: {
-  "Logs" => "/logs",
-  "Log #{@log.id}" => "/lettings-logs/#{@log.id}",
-  "Review lettings log" => "",
-}) %>
+<% if @log.is_a? SalesLog %>
+  <% content_for :title, "Review sales log" %>
+  <% content_for :breadcrumbs, govuk_breadcrumbs(breadcrumbs: {
+    "Logs" => "/logs",
+    "Log #{@log.id}" => "/sales-logs/#{@log.id}",
+    "Review sales log" => "",
+  }) %>
+<% else %>
+  <% content_for :title, "Review lettings log" %>
+  <% content_for :breadcrumbs, govuk_breadcrumbs(breadcrumbs: {
+    "Logs" => "/logs",
+    "Log #{@log.id}" => "/lettings-logs/#{@log.id}",
+    "Review lettings log" => "",
+  }) %>
+<% end %>
```

Comment:
```suggestion
<% if @log.sales? %>
  <% content_for :title, "Review sales log" %>
  <% content_for :breadcrumbs, govuk_breadcrumbs(breadcrumbs: {
    "Logs" => "/logs",
    "Log #{@log.id}" => "/sales-logs/#{@log.id}",
    "Review sales log" => "",
  }) %>
<% else %>
  <% content_for :title, "Review lettings log" %>
  <% content_for :breadcrumbs, govuk_breadcrumbs(breadcrumbs: {
    "Logs" => "/logs",
    "Log #{@log.id}" => "/lettings-logs/#{@log.id}",
    "Review lettings log" => "",
  }) %>
<% end %>
```

We already have `Log#lettings?` and `LettingsLog#lettings?` — could you create the equivalent?